subprojects/docs/src/docs/stylesheets/userGuideHtml.xsl

```diff
@@ -171,10 +171,25 @@
                                     <li><a href="/userguide/feature_lifecycle.html">Feature Lifecycle</a></li></ul></li>
                             <li><a class="nav-dropdown" data-toggle="collapse" href="#managing-dependencies" aria-expanded="false" aria-controls="managing-dependencies">Managing Dependencies</a>
                                 <ul id="managing-dependencies">
-                                    <li><a href="/userguide/artifact_dependencies_tutorial.html">Dependency Management Basics</a></li>
-                                    <li><a href="/userguide/dependency_management.html">Advanced Dependency Management</a></li>
-                                    <li><a href="/userguide/dependency_types.html">Dependency Types</a></li>
-                                    <li><a href="/userguide/repository_types.html">Repository Types</a></li></ul></li>
+                                    <li><a class="nav-dropdown" data-toggle="collapse" href="#dependency-management-tutorials" aria-expanded="false" aria-controls="dependency-management-tutorials">Tutorials</a>
+                                        <ul id="dependency-management-tutorials">
+                                            <li><a href="/userguide/artifact_dependencies_tutorial.html">Dependency Management for Java Projects</a></li>
+                                        </ul>
+                                    </li>
+                                    <li><a href="/userguide/dependency_management_introduction.html">Introduction</a></li>
```

Comment:
Consider "Introduction to Dependency Management" perhaps.

Reply:
done.